pom.xml

```diff
@@ -105,6 +105,11 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.aether</groupId>
```

Comment:
Confirmed. I'm not happy with the aether stuff in maven 3.1.0, but I think we're stuck with what we're stuck with.

Hopefully this will work correctly in 3.1.1 or later... 🎱  tells me we'll have more (or different) bug reports soon.